packages/playback/src/lib/consts/eme-robustness.ts

```diff
@@ -0,0 +1,7 @@
+export enum RobustnessLevel {
+  'SW_SECURE_CRYPTO',
+  'SW_SECURE_DECODE',
+  'HW_SECURE_CRYPTO',
+  'HW_SECURE_DECODE',
+  'HW_SECURE_ALL',
+}
```

Comment:
these are widevine-specific :)

Reply:
You're totally right. Deleting this and reverting to type string